docs/examples/page-header/additional-sections.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <el-page-header :icon="null">
+  <el-page-header :icon="undefined">
```

Comment:
`undefined` will not take effect. So don't change it.

Reply:
Don't want to change it? ´null´will cause a type error

Reply:
Has been changed back to null.
https://github.com/element-plus/element-plus/pull/16370/commits/279305a514de5f212929a1ff5a3a25aabe2e7d0c

Reply:
It's an error of prop types. May be fixed later.